libraries/api/src/middleware/schema.js

```diff
@@ -64,6 +64,9 @@ const validateSchemas = ({validator, absoluteSchemas, rootUrl, serviceName, entr
     // Add a reply method sending JSON replies, this will always reply with HTTP
     // code 200... errors should be sent with res.json(code, json)
     res.reply = (json) => {
+      if (res.headersSent) {
+        monitor.reportError(new Error('called send twice'));
```

Comment:
I think you'll want to `return` here, otherwise Express throws its *own* error ("Cannot set headers after they are sent to the client").

Reply:
Also, let's use a more verbose error message: "API method implementation called res.send twice"